### DISCUSSION TITLE
Cloudflare Workers and accessing waitUntil ?

### DISCUSSION THREAD

**Discussion by danboyle8637** [honojs/hono#1619] [Q&A]
I'm using Hono with Cloudflare Workers. With "normal" workers you get access to a context parameter where you can `waitUntil` ... pretty much do some async work while returning a response immediately. 

Is there a way to get access to this with Hono? I can't see to find anything in the docs... nor just messing around with the code. Any help?

**Comment by yusukebe** [comment]
https://hono.dev/api/context#executionctx

**Comment by danboyle8637** [comment]
Nice!!! @yusukebe Thank you. I have no idea how I missed that. I appreciate it.
